typescript/phoenix-sdk/config.json

```diff
@@ -24,12 +24,24 @@
         "symbol": "mSOL",
         "mint": "7KowotNADmz2krLdRQKBWygyG4Rsvtne4EG7WU7MBQd8",
         "logoUri": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/mSoLzYCxHdYgdzU16g5QSh3i5K3z3KZK7ytfqcJm7So/logo.png"
+      },
+      {
+        "name": "Wrapped SOL",
+        "symbol": "wSOL",
```

Comment:
is this the correct name? perhaps we should rename our fake wSOL to something else?

everywhere on mainnet (including the old token config) this wrapped sol has symbol "SOL" so I would prefer to be consistent there

Reply:
Yeah this one is pretty annoying, I'm indifferent here. I think ideally this shows the Drift logo in the market selection modal, but it will just show a duplicate SOL

Reply:
Can I merge to get the testing to work?